project/PlayArtifact.scala

```diff
@@ -1,48 +0,0 @@
-package com.gu.deploy
```

Comment:
Is there any reason to keep this empty file?

Reply:
No reason at all, just missed it. Fixed now.